cmd/benchmark/main.go

```diff
@@ -124,11 +123,22 @@ func newTransferDifferentMemberScenarios(out io.Writer, insSDK *sdk.SDK, members
 	}
 }
 
-func newCreateMemberScenarios(out io.Writer, insSDK *sdk.SDK, members []*sdk.Member, concurrent int, repetitions int) benchmark {
+func newCreateMemberScenarios(out io.Writer, insSDK *sdk.SDK, concurrent int, repetitions int) benchmark {
 	return benchmark{
-		scenario: &createMembersScenario{
-			insSDK:  insSDK,
-			members: members,
+		scenario: &createMemberScenario{
+			insSDK: insSDK,
+		},
+		concurrent:  concurrent,
+		repetitions: repetitions,
+		name:        "CreateMember",
+		out:         out,
+	}
+}
+
+func newMigrationScenarios(out io.Writer, insSDK *sdk.SDK, concurrent int, repetitions int) benchmark {
+	return benchmark{
+		scenario: &MigrationScenario{
+			insSDK: insSDK,
 		},
 		concurrent:  concurrent,
 		repetitions: repetitions,
```

Comment:
below this line name param must be "Migration", not "CreateMember"

Reply:
Done